pkg/kwokctl/components/etcd.go

```diff
@@ -36,32 +36,30 @@ type BuildEtcdComponentConfig struct {
 
 // BuildEtcdComponent builds an etcd component.
 func BuildEtcdComponent(conf BuildEtcdComponentConfig) (component internalversion.Component, err error) {
+	exposePeerPort := true
```

Comment:
nit: Maybe we should add a flag for this

Reply:
I think that could be tricky for following reasons:
- On binary runtime, you would have to have port available if I am not mistaking
- On other runtimes, if you specify --etcd-port, it is going to be exposed, but under the hood, it is still going to use 2379, so basically, by specifying the flag you are exposing it... 
If flag is added to expose, then on docker and kind runtimes, your --etcd-port is useless if --expose-port or similar flag is not set
Please correct me if I am wrong :smile:

Reply:
Sorry for the misunderstanding caused by my comment, I meant whether to add `--etcd-peer-port` to expose the peer port.

Reply:
Oh, sure, I can do that :relaxed: on it

Reply:
NVM the comment, peer port is used for etcd clusters, but should not be needed in our kwok usage scenario.